.scripts/before_deploy.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+rm -rf vendor
+composer install --no-dev
+
+zip -r ebanx-payment-gateway-for-woocommerce . -x "*.git*"
```

Comment:
here too.

Reply:
You should use TRAVIS env's to access directories, something like: `TRAVIS_BUILD_DIR`